runtime/src/bank.rs

```diff
@@ -2009,10 +2008,19 @@ mod tests {
             system_transaction::transfer(&mint_keypair, &key1.pubkey(), 1, genesis_block.hash());
         assert_eq!(bank.process_transaction(&tx_transfer_mint_to_1), Ok(()));
         assert_eq!(bank.is_delta.load(Ordering::Relaxed), true);
+
+        let bank1 = new_from_parent(&bank);
+        assert_eq!(bank1.is_delta.load(Ordering::Relaxed), false);
+        assert_eq!(bank1.hash_internal_state(), bank.hash());
+        // ticks don't make a bank into a delta
+        bank.register_tick(&Hash::default());
```

Comment:
@rob-solana , should this be `bank1`, since the `is_delta` check on the next line is on `bank1`?

Reply:
yep, good catch